templates/zerver/navbar.html

```diff
@@ -2,7 +2,7 @@
 <nav class="header-main rightside-userlist" id="top_navbar">
   <div class="column-left">
       <a class="no-style brand" href="#">
-          <div class="nav-logo"></div>
+          <img src="../../static/images/logo/zulip-icon-128x128.png"/>
```

Comment:
I haven't tested, but from triangulating from the rest of the html files in `templates/zerver`, `"/static/images/..."` should suffice.